### PR TITLE
safety_signing

### PR DESCRIPTION
**Pull request by ts24github**
## Type of change

- [ ] Add a new logo or icon for a new core integration
- [ ] Add a missing icon or logo for an existing core integration
- [x] Add a new logo or icon for a custom integration (custom component)
  - [x] I've opened up a PR for my custom integration on the [Home Assistant
    Python wheels repository](https://github.com/home-assistant/wheels-custom-integrations)
- [ ] Replace an existing icon or logo with a higher quality version
- [ ] Removing an icon or logo

## Additional information

- This PR fixes or closes issue: fixes #
- Link to code base pull request: 
- Link to documentation pull request: 
- Link to integration documentation on our website: 

## Checklist

- [x] The added/replaced image(s) are **PNG**
- [x] Icon image size is 256x256px (`icon.png`)
- [x] hDPI icon image size is 512x512px for  (`icon@2x.png`)
- [x] Logo image size has min 128px, but max 256px, on the shortest side (`logo.png`)
- [x] hDPI logo image size has min 256px, but max 512px, on the shortest side (`logo@2x.png`)

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @ts24github, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;brands#3240).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by ts24github** [comment]
logo
logo@2x
icon
icon@2x

File: custom_integrations/safety_signing~d8472769932447d040563af20b7e54fac54887df [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (frenck) [inline custom_integrations/safety_signing~d8472769932447d040563af20b7e54fac54887df:1]: This file doesn't belong here? Please remove it.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Hi there @ts24github 👋 

Could you point me to a resource providing more information about the integration/product/manufacturer of this integration?

../Frenck

**Comment by ts24github** [comment]
> Hi there @ts24github 👋
> 
> Could you point me to a resource providing more information about the integration/product/manufacturer of this integration?
> 
> ../Frenck

Yes,
-This integration about USB token for KPI.
-Auto control device character via usb token and HSM
through the API connect to my server

**Comment by frenck** [comment]
Cool, but could you point me to a resource providing more information about the integration/product/manufacturer of this integration?

**Comment by ts24github** [comment]
> Cool, but could you point me to a resource providing more information about the integration/product/manufacturer of this integration?


Yes of course,
-This is my website : www.safetysigning.vn
-About manufacturer : 
SAFEcert Corp
X-04.77, North Towers Building, Sunrise City, 27 Nguyen Huu Tho, Tan Hung Ward, District 7, Ho Chi Minh City, Vietnam
Phone: (028)-668-23732
Email: info@safecert.com.vn

**Comment by ts24github** [comment]
> Cool, but could you point me to a resource providing more information about the integration/product/manufacturer of this integration?
Manufacturer Devices by : 
FEITIAN Technologies Co., Ltd
www.ftsafe.com

**Review by frenck — APPROVED** [review]
Alright! thanks, @ts24github 👍

**Pull request merged**